### ISSUE TITLE
Question about Training Details

### ISSUE THREAD

**Issue by liu6381810** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#18]
Hi, Thanks for you good work!
Now I'm rewriting this project by keras
But I have some question about training 
When I generate the ground truth 
the confidence map's size is 46 * 46
But only a few points set to values > 0 (because only the points near key point is positive)
and all other points set to 0
Then when I try to training the network 
I found it tends to predict all the points to be zero
and even so the loss is low because we use squared error 
So is there any advice for training the network to avoid this problem?

**Comment by tensorboy** [comment]
try `train.py`

**Comment by jjjkkkjjj** [comment]
@liu6381810 
This conversation is too old. But did you remember how to resolve that problem which a model predicts all the points to be zero?
I really want to know a solution...
